activesupport/lib/active_support/process_time.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module ProcessTime # :nodoc:
```

Comment:
Does it make sense to name this `ProcessClock`?

Reply:
Yeah, seems It'll be better to name it `ProcessClock`